src/main/resources/messages-form-flow.properties

```diff
@@ -108,3 +108,10 @@ address.street-address-2=Street address line 2
 address.city=City
 address.state=State
 address.zip=ZIP code
+#
+language-preferences.button=Translate
+language-preferences.preview.first=Espa\u00F1ol
+language-preferences.preview.second=
```

Comment:
Yes! The idea is, from the design there can be up to two languages shown in the button. For the starter app, since we're only supporting two languages, we only show a preview of one language.

For other apps, if they want to show the second language, they can override this property in their own `messages`.

<img width="256" alt="Screenshot 2023-07-10 at 12 24 33 PM" src="https://github.com/codeforamerica/form-flow/assets/9101728/5c64362b-5180-4a16-8bcc-d0b2331154bc">